### ISSUE TITLE
Broken clprf integration tests

### ISSUE THREAD

**Issue by lintool** [castorini/pyserini#841]
At commit 56527181dc70226fd57194dbe3f322e56594269c:

```
$ python -m unittest integrations.clprf.test_clprf.TestSearchIntegration.test_core18_lr
Running core18 topics, saving to ./integrations/tmp4232/core18_lr.txt...
100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 110/110 [00:02<00:00, 46.09it/s]
 53%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████                                                                                                                         | 533/1000 [00:11<00:10, 45.10it/s]
  0%|                                                                                                                                                                                                                                                                               | 0/50 [00:14<?, ?it/s]
Traceback (most recent call last):
  File "/opt/local/stow/Python3-3.6.4/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/opt/local/stow/Python3-3.6.4/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/scratch2/jimmylin/pyserini/pyserini/search/__main__.py", line 260, in <module>
    scores, docids = ranker.rerank(docids, scores)
  File "/scratch2/jimmylin/pyserini/pyserini/search/reranker.py", line 85, in rerank
    return self._rerank_with_classifier(doc_ids, search_scores)
  File "/scratch2/jimmylin/pyserini/pyserini/search/reranker.py", line 68, in _rerank_with_classifier
    train_vecs, train_labels, test_vecs = self._get_prf_vectors(doc_ids)
  File "/scratch2/jimmylin/pyserini/pyserini/search/reranker.py", line 63, in _get_prf_vectors
    test_vecs = self.vectorizer.get_vectors(doc_ids)
  File "/scratch2/jimmylin/pyserini/pyserini/vectorizer/_base.py", line 127, in get_vectors
    tfidf = tf[term] * self.idf_[term]
TypeError: unsupported operand type(s) for *: 'NoneType' and 'float'
F
======================================================================
FAIL: test_core18_lr (integrations.clprf.test_clprf.TestSearchIntegration)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch2/jimmylin/pyserini/integrations/clprf/test_clprf.py", line 334, in test_core18_lr
    self.assertEqual(status, 0)
AssertionError: 256 != 0

----------------------------------------------------------------------
Ran 1 test in 54.713s

FAILED (failures=1)
```

**Comment by lintool** [comment]
Update: I just finished running the following on my iMac Pro:

```
python -m unittest discover -s integrations/clprf
```

Amazingly, all 44 tests passed in ~6.3 hours. I said to myself, that can't be right, since previously, the same test suite took ~26.1h. Except that the result appears to be correct!

Dropping the `JString` wrapping increases performance also because we're not needlessly creating a bunch of objects in the inner loop. Yay!

Just for confirmation, I ran:

```
python -m unittest integrations.clprf.test_clprf.TestSearchIntegration.test_robust05_lr
```

This is a test that does suffer from the emoji problem:

+ At 412ebb5bc2f1db200034c2a4504068b2b08f1f26, after the `JString` wrapping removal, test takes ~200s.
+ At 5469a399fe77504bfd6043cc7400746e97754659, with the `JString' wrapping, test takes ~660s.

Wow!
